src/coreclr/inc/bbsweep.h

```diff
@@ -293,15 +202,6 @@ class BBSweep
             hTerminationEvent    = ::WszCreateEvent(pSecurityAttributes, true, false, NULL);
             swprintf_s(objectName, MAX_LONGPATH, W("%sBBSweep_hProfWriterSemaphore"), objectNamePrefix);
             hProfWriterSemaphore = ::WszCreateSemaphore(pSecurityAttributes, MAX_COUNT, MAX_COUNT, objectName);
-
-#ifndef FEATURE_CORESYSTEM // @CORESYSTEMTODO
-cleanup:
-            if (pSD) delete [] ((char *) pSD);
-            if (pAdminSid) FreeSid(pAdminSid);
-            if (hToken) CloseHandle(hToken);
-            if (pACL) LocalFree(pACL);
```

Comment:
nit: we can delete `#pragma push_macro("LocalFree")` and `#pragma pop_macro("LocalFree")` lines from this file as well (since this was the only usage).